### PR TITLE
refactor(rxAuth): rx-prefixes update Auth

### PR DESCRIPTION
**Pull request by cathysiller**
JIRA: https://jira.rax.io/browse/FRMW-1147

### LGTMs
- [x] Dev LGTM
- [x] Dev+Vidyo LGTM

### CODE REVIEW

File: src/utilities/rxAuth/scripts/rxAuth.spec.js [hunk 1]
```diff
@@ -10,7 +10,7 @@ describe('utilities:Auth', function () {
                     'id': '655062',
                     'name': '655062'
                 },
-                'RAX-AUTH:authenticatedBy': ['PASSWORD']
+                'RAX-rxAuth:authenticatedBy': ['PASSWORD']
```
Comment (CITguy) [inline src/utilities/rxAuth/scripts/rxAuth.spec.js]: This needs to stay as `RAX-AUTH:authenticatedBy`. Please revert.

Comment (cathysiller) [inline src/utilities/rxAuth/scripts/rxAuth.spec.js]: reverted !!

File: src/utilities/rxAuth/scripts/rxAuth.spec.js [hunk 2]
```diff
@@ -10,7 +10,7 @@ describe('utilities:Auth', function () {
                     'id': '655062',
                     'name': '655062'
                 },
-                'RAX-AUTH:authenticatedBy': ['PASSWORD']
+                'RAX-Auth:authenticatedBy': ['PASSWORD']
```
Comment (CITguy) [inline src/utilities/rxAuth/scripts/rxAuth.spec.js]: Please revert to original capitalization. `RAX-AUTH`

Comment (cathysiller) [inline src/utilities/rxAuth/scripts/rxAuth.spec.js]: reverted to capitalization -- missed the first time :)

File: src/utilities/rxAuth/scripts/rxAuth.spec.js [hunk 3]
```diff
@@ -0,0 +1,179 @@
...
+            session.logout = sinon.stub();
+            session.isCurrent = sinon.stub().returns(true);
+            session.isAuthenticated = sinon.stub().returns(true);
+            permission.getRoles = sinon.stub().returns([{ 'name': 'admin' }]);
+
+            auth = $injector.get('Auth');
+        });
+    });
+
+    describe('service:rxAuth', function () {
```
Comment (cathysiller) [inline src/utilities/rxAuth/scripts/rxAuth.spec.js]: update to Auth

**Comment by parlarjb** [comment]
LGTM

**Comment by halkon** [comment]
LGTM pending update

**Pull request merged**